openidm-zip/src/main/resources/startup.sh

```diff
@@ -32,12 +32,21 @@ abspath() {
     fi
 }
 
-JAVA_VER=$(java -version 2>&1 | sed 's/.* version "\(.*\)\.\(.*\)\..*"/\1\2/; 1q')
+JAVA_VER=$(java -version 2>&1 | sed 's/.* version "\(.*\)\.\(.*\)\..*".*/\1\2/; 1q')
 if [ "$JAVA_VER" -lt 17 ]; then
   echo "Java version 1.7 or higher required";
   exit 1;
 fi
 
+# Enable reflective access to classloader for newer JDK versions
```

Comment:
Is there any specific issue in Felix's issue tracker for this workaround? If so, please add reference here.

Reply:
The issue is here: https://issues.apache.org/jira/browse/FELIX-5765

This workaround is mentioned by the third comment